src/components/aboutUs/components/styles.css

```diff
@@ -0,0 +1,43 @@
+.content {
+    background: white;
+}
+.top-div {
+    background: white; 
+    text-align: center;
+}
+.top-text {
+    text-align: center;
+    font-size: 18px;
+    font-weight: 500;
+    font-family:  'Geneva', Verdana;
+}
+.top-text:first-letter {
+    font-size: 25px;
+    font-style: italic;
+}
+.top-card {
+    margin-left: 23%;
+    margin-top: 4%; 
+    padding-top: 5%;
+    padding-bottom: 2%;
+    width: 52%;
+}
+.team-card {
+    position: relative;
+    left: 45%;
+    width: 100%;
+    text-align: center; 
+    margin-top: 8%;
+}
+.avatar { 
+    width: 55%;
+    height: 50%;
+    margin-left: 23%;
+    margin-right: 50%;
+    display: block;
+}
+.team-header {
+    text-align: center;
+    font-size: 23px;
+    margin: 2%;
+}
```

Comment:
There's additional line at the end of the file

Reply:
how do u mean?

Reply:
ok